### ISSUE TITLE
Crosswalk mapping of ONIX list 196 code 09

### ISSUE THREAD

**Issue by jonaslil** [w3c/publ-a11y#757]
The Accessibility Properties Crosswalk maps ONIX list 196 code 09 (Inaccessible, or known limited accessibility) to accessibilityFeature none.

The Schema → ONIX mapping seems fine, but the reverse direction (ONIX → Schema) appears problematic.

A publication marked as generally inaccessible or having limited accessibility may still have accessibility features. For example, a publication may lack all image descriptions but still have navigation related features, display transformability, or a well-defined reading order.

A structural difference between the two is that ONIX code 09 does not, as far as I know, exclude using other codes from codelist 196 to describe accessibility features, while in Schema, none explicitly excludes declaring positive features.

So applying the ONIX → Schema mapping suggested by the crosswalk might result in a publication having accessibilityFeature none as well as other positive values, which is prohibited.

This issue came up in discussions with the Finnish National Library. They need to map publishers' ONIX records to MARC 21 using Schema values. If two values appear on the same line in the crosswalk tables, they assume the values are fully mappable to one another.

I'm not sure what to propose as a solution here, but any limitations to the mappings proposed in the Crosswalk need to be explicitly stated.
